pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1411,7 +1411,16 @@ func (n *NetworkPolicyController) getAppliedToWorkloads(g *antreatypes.AppliedTo
 		if found {
 			grp := group.(*antreatypes.Group)
 			pods, ees, err := n.getInternalGroupWorkloads(grp)
-			return pods, ees, nil, err
+			var nodes []*v1.Node
+			if grp.Selector != nil {
+				if grp.Selector.NodeSelector != nil {
```

Comment:
nodeSelector is exclusive, right? then should it check `grp.Selector.NodeSelector` first and skips `getInternalGroupWorkloads` which has some cost?